pulser-core/pulser/sequence/_seq_drawer.py

```diff
@@ -487,6 +412,276 @@ def phase_str(phi: float) -> str:
             special_kwargs = dict(labelpad=10) if i == 0 else {}
             ax.set_ylabel(LABELS[i], fontsize=14, **special_kwargs)
 
+        if draw_target_regions:
```

Comment:
I don't think it is an option to display it.

Reply:
The "optional" is because if we are drawing a sequence, we will draw it in `draw_sequence` instead of drawing in `draw_samples`

Reply:
ok